packages/override-tools/src/test/Manifest.test.ts

```diff
@@ -16,35 +16,35 @@ import {
 
 const reactFiles: Array<MockFile> = [
   {
-    filename: 'aaa/aaa.js',
+    filename: 'aaa\\aaa.js',
```

Comment:
Changing everything to backslash just for consistency since the world right now is a bit confusing.